rep-0149.rst

```diff
@@ -914,6 +888,18 @@ each package.  See REP-0134 for details.
 Further build types may eventually be defined, such as: "make",
 "autotools", "rosbuild", or "custom".
 
+.. raw:: html
+
+  <font color="red">
+
+Attributes
+..........
+
+ The `condition` attribute as defined for `\<build_depend\> (multiple)`_.
```

Comment:
With the additional attribute the `build_type` needs to be defined as "(multiple)" now to allow more than one to exist.

Reply:
Multiple, but only allowing one to be active after conditions are evaulated in 5966d1b1c3d96b690a5ecbf27713072c19a28b2a

Reply:
Before we never did but maybe it is time to specify which one should be used in case multiple tags are "active". First or last?

Reply:
> The last `<build_type>` encountered is the last to set the build type.

Sounds good to me. If that build type is not supported by the tool "bad luck" :wink: